projects/plugins/jetpack/extensions/blocks/slideshow/view.js

```diff
@@ -41,6 +41,7 @@ if ( typeof window !== 'undefined' ) {
 						onlyInViewport: true,
 					},
 					pagination: {
+						el: '.swiper-pagination',
```

Comment:
Note: this is added so that we can do a shallow merge instead of a deep merge in `create-swiper.js`.